client/my-sites/patterns/components/get-access-modal/style.scss

```
@@ -0,0 +1,111 @@
+@import "calypso/my-sites/patterns/mixins";
+@import "@wordpress/base-styles/breakpoints";
+@import "@automattic/components/src/styles/typography";
+
+.patterns-get-access-modal {
```

```suggestion
.patterns-sign-up-modal {
```
Optional nit, but I think this name is easier to understand in the code, even if we use `Get access` in the actual button title.

---

It's not only "Sign up", it's also "Sign in", so I wanted to use some general name and `get-access` was looking quite good (IMO). But I don't have strong objections.